crates/teloxide/examples/chat_member_updates.rs

```diff
@@ -0,0 +1,96 @@
+use teloxide::{
+    dispatching::Dispatcher,
+    prelude::*,
+    types::{ChatMemberUpdated, ParseMode, Update},
+    utils::html,
+};
+use teloxide_core::adaptors::DefaultParseMode;
+
+/// We need the Bot alias type because we need use the
+/// `ParseMode::Html` by default
+/// (This is optional for this example, use the `parse_mode` method
+/// for set the parse mode in a specific message)
```

Comment:
```suggestion
/// We use a type alias to be able to write just `bot: Bot` in handlers, instead of a lengthy `DefaultParseMode<Bot>`.
///
/// `DefaultParseMode` here is a [requester adaptor], that allows us to specify a default parse mode. The other option would be to use `.parse_mode(...)` after each API call manually (e.g. `bot.send_message(...).parse_mode(...).await?`).
///
/// [requester adaptor]: teloxide::requests::Requester#adaptors
```